src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -72,11 +72,11 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
                 continue;
             }
 
-            String pathAdjusted = file.absolutePath().replace('\\', '/');
+            String pathAdjusted = file.absolutePath().replace('\\', '/').toLowerCase();
```

Comment:
So I can only get tslint to return the same casing as I supply it, i.e. if I give it a file with a lower-case drive letter it spits it back lower-case, while if I give it one upper-case it appeared to keep it upper-case. Will try and repro tonight